code/game/gamemodes/miniantags/guardian/types/healer.dm

```diff
@@ -90,21 +90,23 @@
 	set desc = "Mark a floor as your beacon point, allowing you to warp targets to it. Your beacon will not work in unfavorable atmospheric conditions."
 	if(beacon_cooldown < world.time)
 		var/turf/beacon_loc = get_turf(loc)
-		if(isfloorturf(beacon_loc))
-			var/turf/simulated/floor/F = beacon_loc
-			F.icon = 'icons/turf/floors.dmi'
-			F.name = "bluespace recieving pad"
-			F.desc = "A recieving zone for bluespace teleportations. Building a wall over it should disable it."
-			F.icon_state = "light_on"
-			to_chat(src, "<span class='danger'>Beacon placed! You may now warp targets to it, including your user, via Alt+Click. </span>")
+		if(isfloorturf(beacon_loc) && !islava(beacon_loc) && !ischasm(beacon_loc))
 			if(beacon)
-				beacon.ChangeTurf(/turf/simulated/floor/plating)
-			beacon = F
+				qdel(beacon)
```

Comment:
```
			QDEL_NULL(beacon)
```
Can do this instead of line 94 and 95. I can't select that range though due to how the diff is set up.